app/jobs/sync_salesforce_opportunities_job.rb

```diff
@@ -0,0 +1,38 @@
+class SyncSalesforceOpportunitiesJob < ApplicationJob
+  queue_as :default
+
+  def perform(id=nil)
+    if id
+      sf_opportunities = OpenStax::Salesforce::Remote::Opportunity.where(id:id)
```

Comment:
same as above

instead of `id`, if you make this `contact_id`, you can update the opps only for this user - which would be better on their first visit

Reply:
Done. Updated BaseController to send a contact_id as well.